solutions/documentation.md

```diff
@@ -0,0 +1,55 @@
+# Recommended solutions for Documentation
+
+Patterns for serving documentation pages to our communities and clients.
+
+## Community documentation
+
+The majority of our documentation will be community-focused. This means both that the various communities around our products are using the documentation, and that we hope they will also help to improve the documentation.
+
+### Discourse forums for content
+
+We recommend that community documentation content should live in [Discourse forums](https://www.discourse.org/). Discourse forums provide a useful model for building a community around our content.
+
+This content could live inside a category inside [discourse.ubuntu.com](https://discourse.ubuntu.com/) or any other Discourse installation.
+
+#### Structure
+
+Documentation content should be stored in ["wiki" posts](https://meta.discourse.org/t/what-is-a-wiki-post/30801) that users can comment on, and when they build up enough trust, can edit directly. The [trust levels](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/) have configurable thresholds. Documentation topics should be grouped under an appropriately-named [category](https://meta.discourse.org/t/how-to-add-categories/71859) - e.g. `docs` or `product-docs`.
+
+Each documentation set shuold have an introductory page. At the bottom of this page should be a horizontal rule, followed by the heading "Content", and then a navigation list which should link to other documentation pages in the forum, e.g.:
+
+``` markdown
+Hello. Welcome to our documentation.
+
+Please spend a while here.
+
+---
+
+# Content
+
+- [Getting started](/t/getting-started/3876)
+- [Configuration options](/t/configuration-options/87)
```

Comment:
Can we show a small example of one with sectioned navigation? How would that work?

Reply:
The application I'm writing isn't opinionated about this. All it will do is convert links so they'll work on the website - other than that, it will simply parse anything below `# Content` faithfully, and output the markup into a `<nav>` section or whatever.

If we are only planning so support certain navigation structures from a styling (Vanilla) perspective, then it would make sense to explain those structures here, but I'm not the right person to write that. So how about I merge this first and then you augment this section with the navigation structures you plan to support?

Reply:
After discussing with voices, we've agreed I'll add a couple of sections, as this is the structure most documentations sets will have.